packages/stockflux-news/src/App.js

```diff
@@ -91,7 +91,22 @@ function App() {
   return (
     <div className="stockflux-news">
       <Components.Titlebar />
-      <div className="header">{name ? name : symbol} News</div>
+      <div className="header">
+        {name ? name : symbol} News
+        <div className="icons">
+          {' '}
```

Comment:
Is this `{' '}` intentional? If this is to ensure the space between text and buttons, maybe we should add `margin-left` instead?

Reply:
This is a mistake, think it was added by the linter and left in instead of being removed